config/config.ts

```diff
@@ -105,11 +105,13 @@ const config: Configuration = {
                             url: 'https://flybywirecdn.com/addons/a32nx/master',
                             alternativeUrls: [
                                 'external/a32nx/master',
-                                // move old experimental users over to dev
-                                'https://cdn.flybywiresim.com/addons/a32nx/cfbw-cap',
-                                'https://cdn.flybywiresim.com/addons/a32nx/cfbw',
```

Comment:
Any reason to remove this? Just in case some old installs are still around somewhere - it doesn't seem like it would hurt to keep this around.

Reply:
It’s so old I just like to clean up occasionally. But sure can add it again